hacking/env-setup

```diff
@@ -59,7 +59,7 @@ fi
 (
 	cd "$ANSIBLE_HOME"
 	if [ "$verbosity" = silent ] ; then
-	    gen_egg_info > /dev/null 2>&1
+	    gen_egg_info > /dev/null 2>&1 &
             find . -type f -name "*.pyc" -exec rm -f {} \; > /dev/null 2>&1
 	else
 	    gen_egg_info
```

Comment:
From my original description above:

> I've only added it for -q since you wouldn't see the message anyway. Without -q we don't background the generation.

This is because gen_egg_info prints to the screen.  If you background that, you'll get weird text printing to the terminal out of sync with anything else that is happening.